### ISSUE TITLE
Bug: Problem with surfshark in the latest docker update

### ISSUE THREAD

**Issue by rachid598** [qdm12/gluetun#429]
**Is this urgent?**: No

**Host OS** (approximate answer is fine too): Ubuntu 18

**CPU arch** or **device name**: amd64

**What VPN provider are you using**:

**What are you using to run your container?**: Docker

**What is the version of the program** (See the line at the top of your logs)

```
Running version latest built 
```

**What's the problem** 🤔
Since i update today to the latest i can't connect with surfshark.

**Share your logs... (careful to remove in example tokens)**

```
2021/04/19 17:32:57 INFO storage: merging by most recent 7378 hardcoded servers and 7378 servers read from /gluetun/servers.json
2021/04/19 17:32:57 INFO routing: default route found: interface eth0, gateway 172.23.0.1
2021/04/19 17:32:57 INFO routing: local ethernet link found: gretap0
2021/04/19 17:32:57 INFO routing: local ethernet link found: erspan0
2021/04/19 17:32:57 INFO routing: local ethernet link found: eth0
2021/04/19 17:32:57 INFO routing: local ipnet found: 172.23.0.0/16
2021/04/19 17:32:57 INFO routing: default route found: interface eth0, gateway 172.23.0.1
2021/04/19 17:32:57 INFO routing: adding route for 0.0.0.0/0
2021/04/19 17:32:57 INFO firewall: firewall disabled, only updating allowed subnets internal list
2021/04/19 17:32:57 INFO routing: default route found: interface eth0, gateway 172.23.0.1
2021/04/19 17:32:57 INFO openvpn configurator: checking for device /dev/net/tun
2021/04/19 17:32:57 WARN TUN device is not available: open /dev/net/tun: no such file or directory
2021/04/19 17:32:57 INFO openvpn configurator: creating /dev/net/tun
2021/04/19 17:32:57 INFO firewall: enabling...
2021/04/19 17:32:57 INFO firewall: enabled successfully
2021/04/19 17:32:57 INFO healthcheck: listening on 127.0.0.1:9999
2021/04/19 17:32:57 INFO dns over tls: using plaintext DNS at address 1.1.1.1
2021/04/19 17:32:57 INFO http server: listening on 0.0.0.0:8000
2021/04/19 17:32:57 INFO firewall: setting VPN connection through firewall...
2021/04/19 17:32:57 INFO openvpn configurator: starting openvpn
2021/04/19 17:32:57 INFO openvpn: OpenVPN 2.5.1 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 16 2021
2021/04/19 17:32:57 INFO openvpn: library versions: OpenSSL 1.1.1k  25 Mar 2021, LZO 2.10
2021/04/19 17:32:57 WARN openvpn: --ping should normally be used with --ping-restart or --ping-exit
2021/04/19 17:32:57 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]5.253.205.181:1194
2021/04/19 17:32:57 INFO openvpn: UDP link local: (not bound)
2021/04/19 17:32:57 INFO openvpn: UDP link remote: [AF_INET]5.253.205.181:1194
2021/04/19 17:32:57 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1633', remote='link-mtu 1581'
2021/04/19 17:32:57 WARN openvpn: 'auth' is used inconsistently, local='auth SHA512', remote='auth [null-digest]'
2021/04/19 17:32:57 INFO openvpn: [be-bru-v014.prod.surfshark.com] Peer Connection Initiated with [AF_INET]5.253.205.181:1194
2021/04/19 17:33:03 INFO openvpn: OPTIONS ERROR: failed to negotiate cipher with server.  Add the server's cipher ('AES-256-GCM') to --data-ciphers (currently 'AES-256-CBC') if you want to connect to this server.
2021/04/19 17:33:03 INFO openvpn: ERROR: Failed to apply push options
2021/04/19 17:33:03 INFO openvpn: Failed to open tun/tap interface
2021/04/19 17:33:03 INFO openvpn: SIGUSR1[soft,process-push-msg-failed] received, process restarting
2021/04/19 17:33:13 WARN openvpn: --ping should normally be used with --ping-restart or --ping-exit
2021/04/19 17:33:13 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]5.253.205.181:1194
2021/04/19 17:33:13 INFO openvpn: UDP link local: (not bound)
2021/04/19 17:33:13 INFO openvpn: UDP link remote: [AF_INET]5.253.205.181:1194
2021/04/19 17:33:14 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1633', remote='link-mtu 1581'
2021/04/19 17:33:14 WARN openvpn: 'auth' is used inconsistently, local='auth SHA512', remote='auth [null-digest]'
2021/04/19 17:33:14 INFO openvpn: [be-bru-v014.prod.surfshark.com] Peer Connection Initiated with [AF_INET]5.253.205.181:1194
2021/04/19 17:33:20 INFO openvpn: OPTIONS ERROR: failed to negotiate cipher with server.  Add the server's cipher ('AES-256-GCM') to --data-ciphers (currently 'AES-256-CBC') if you want to connect to this server.
2021/04/19 17:33:20 INFO openvpn: ERROR: Failed to apply push options
2021/04/19 17:33:20 INFO openvpn: Failed to open tun/tap interface
2021/04/19 17:33:20 INFO openvpn: SIGUSR1[soft,process-push-msg-failed] received, process restarting
2021/04/19 17:33:30 WARN openvpn: --ping should normally be used with --ping-restart or --ping-exit
2021/04/19 17:33:30 INFO openvpn: TCP/UDP: Preserving recently used remote address: [AF_INET]5.253.205.181:1194
2021/04/19 17:33:30 INFO openvpn: UDP link local: (not bound)
2021/04/19 17:33:30 INFO openvpn: UDP link remote: [AF_INET]5.253.205.181:1194
2021/04/19 17:33:30 WARN openvpn: 'link-mtu' is used inconsistently, local='link-mtu 1633', remote='link-mtu 1581'
2021/04/19 17:33:30 WARN openvpn: 'auth' is used inconsistently, local='auth SHA512', remote='auth [null-digest]'
2021/04/19 17:33:30 INFO openvpn: [be-bru-v014.prod.surfshark.com] Peer Connection Initiated with [AF_INET]5.253.205.181:1194
2021/04/19 17:33:36 INFO openvpn: OPTIONS ERROR: failed to negotiate cipher with server.  Add the server's cipher ('AES-256-GCM') to --data-ciphers (currently 'AES-256-CBC') if you want to connect to this server.
2021/04/19 17:33:36 INFO openvpn: ERROR: Failed to apply push options
2021/04/19 17:33:36 INFO openvpn: Failed to open tun/tap interface
2021/04/19 17:33:36 INFO openvpn: SIGUSR1[soft,process-push-msg-failed] received, process restarting
```

**Comment by rachid598** [comment]
I roll back to qmcgaw/gluetun:v3.16.0 and it's fine.
Thanks

**Comment by qdm12** [comment]
You can also set `OPENVPN_CIPHER=AES-256-GCM` on `:latest` and it should work.

Anyway I pushed a change b09147d30f11010ed2dfd16448fe9f46fc7f1e82 to add `AES-256-GCM` to `:latest`, can you try it (without setting `OPENVPN_CIPHER`) see if it works for you?

**Comment by rachid598** [comment]
Hello,
No it's not work with the latest pull.
This is my logs
today at 19:46:22  2021/04/19 19:46:22 INFO firewall: setting VPN connection through firewall...
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn configurator: starting openvpn
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn: OpenVPN 2.5.1 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 16 2021
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn: library versions: OpenSSL 1.1.1k  25 Mar 2021, LZO 2.10
today at 19:46:22  2021/04/19 19:46:22 WARN openvpn: --ping should normally be used with --ping-restart or --ping-exit
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn: Cipher aes-256-cbc:aes-256-gcm not supported
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn: Exiting due to fatal error
today at 19:46:22  2021/04/19 19:46:22 ERROR openvpn: exit status 1
today at 19:46:22  2021/04/19 19:46:22 INFO openvpn: retrying in 15s
today at 19:46:37  2021/04/19 19:46:37 INFO firewall: setting VPN connection through firewall...
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn configurator: starting openvpn
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn: OpenVPN 2.5.1 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 16 2021
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn: library versions: OpenSSL 1.1.1k  25 Mar 2021, LZO 2.10
today at 19:46:37  2021/04/19 19:46:37 WARN openvpn: --ping should normally be used with --ping-restart or --ping-exit
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn: Cipher aes-256-cbc:aes-256-gcm not supported
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn: Exiting due to fatal error
today at 19:46:37  2021/04/19 19:46:37 ERROR openvpn: exit status 1
today at 19:46:37  2021/04/19 19:46:37 INFO openvpn: retrying in 15s

**Comment by qdm12** [comment]
@rachid598 Can you try re-pulling and trying again? I now replaced the default cipher from `aes-256-cbc` to `aes-256-gcm`; Feel free to try with other values using `-e OPENVPN_CIPHER=<somevalue>` if that doesn't work still. Thanks!

**Comment by rachid598** [comment]
Thank you very much it works.
Have a good day

**Comment by qdm12** [comment]
Fixed by 44d8cf9d4e4926d17c6911fb00eb9f1950674742
